Kernel/Arch/riscv64/PCI/Initializer.cpp

```diff
@@ -189,6 +189,12 @@ void initialize()
             }
         }
 
+        // FIXME: 64-bit PCI MMIO regions are very likely above 4G and thus also quite likely after the RAM region(s).
+        //        MemoryManager can't handle accesses to memory after RAM and will panic on such accesses,
+        //        so make the PCI resource allocator only use the (likely smaller) 32-bit region instead.
+        //        The PCI resource allocator will use the 32-bit region if there is no 64-bit region.
+        pci_64bit_mmio_size = 0;
```

Comment:
worth an issue to track this problem?

Reply:
#22720 is about the same underlying issue. The only difference is that SeaBIOS allocates BARs above 4G in that issue, not us.